charts/library/common/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: common
 description: Function library for Helm charts
 type: library
-version: 0.3.0
-kubeVersion: ">=1.16.0-0"
+version: 0.4.0
+kubeVersion: ">=1.19.0-0"
```

Comment:
Having this minimum kubeVersion implies certain other functionality has also been changed/removed which is not the case yet. I have a major refactor/rewrite coming up to take care of this but until then I want to keep the k8s version compatibility intact.

Reply:
Weirdly enough I can't find an exact minimum version that the Gateway API needs, all I found is that they want to support the latest 5 minor versions. Personally I've been running it on 1.22.6 which is much newer than this, but I'll try to find something out about this.